src/apps/stats.ts

```diff
@@ -1,8 +1,8 @@
 import { AnyResponse } from '@octokit/rest'
-import { Request,Response } from 'express'
+import { Request, Response } from 'express'
 
 // Built-in app to expose stats about the deployment
-module.exports = async (app: any): Promise<void> => {
+export = async (app: any): Promise<void> => {
```

Comment:
When calling `import appFn = require('../../src/apps/stats')` in the test, TypeScript complained about this module until I switched from `module.exports =` to `export =`

Reply:
I don't think we have an integration test covering the loading of these apps.  Can you verify that you can use this branch in a new app and it works by doing `npm install macklinu/probot#convert-apps-stats-test`, `npm start`, and visiting localhost:3000/probot/stats

Reply:
I had the same concern and tested it out successfully. But it only works after creating a manifest successfully, since otherwise you're in setup mode.